spec/examples/alter_track_spec.rb

```diff
@@ -25,7 +34,9 @@ class Thing < ActiveRecord::Base
 
   it 'should work with altered column names' do
     thing = Thing.create(:name => 'john')
-    thing.update_attributes(:name => 'john2')
+    thing.name = 'john2'
+    thing.save!
+    # thing.update_attributes(:name => 'john2')
```

Comment:
Can we remove these commented out lines?

I'm ready to merge after this and will bump the version number to correspond with the backward-incompatible changes.

Can we also make an update to the README accordingly?